plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastore.java

```diff
@@ -248,6 +254,27 @@ public void testGetAllTable()
         assertEquals(metastore.getTableNamesStats().getHitRate(), 1.0 / 3);
     }
 
+    @Test
+    public void testBatchGetAllTable()
+    {
+        assertEquals(mockClient.getAccessCount(), 0);
+        assertEquals(metastore.getAllTables(), Optional.of(ImmutableList.of(new SchemaTableName(TEST_DATABASE, TEST_TABLE))));
```

Comment:
nit. Extract variable for schema table name

Reply:
Extracted